data/json/recipes/recipe_armor.json

```diff
@@ -8414,7 +8461,7 @@
   ],
   "components": [
     [
-      [ "duster", 1 ]
+      [ "sleeveless_duster", 1 ]
```

Comment:
Good point.
I was a little skeptical about having a duster in there at all, though. I mean, it's a normal-layer gear, and dusters are outer.